src/utils/__tests__/hasInterpolation-test.js

```diff
@@ -0,0 +1,14 @@
+import hasInterpolation from "../hasInterpolation"
+import test from "tape"
+
+test("isStandardSyntaxValue", t => {
+  t.notOk(hasInterpolation("(min-width#{$value}: 10px)"), "scss interpolation")
+  t.notOk(hasInterpolation("(@{value}min-width : 10px)"), "Less interpolation")
+  t.notOk(hasInterpolation("#{$Attr}-color"), "sass interpolation")
+  t.notOk(hasInterpolation("@{Attr}-color"), "less variable")
+  t.notOk(hasInterpolation("#{50% - $n}"), "SCSS interpolation (id)")
+  t.notOk(hasInterpolation(".n-#{$n}"), "SCSS interpolation (class)")
+  t.notOk(hasInterpolation(":n-#{$n}"), "SCSS interpolation (pseudo)")
+  t.notOk(hasInterpolation(".n-@{n}"), "Less interpolation")
+  t.end()
```

Comment:
All of the above tests should be `ok`, right? Because they all *have interpolation*? Additionally, we need tests for strings that do not have interpolation.